client/curriculum/html/TextElements.tsx

```diff
@@ -11,6 +11,7 @@ import Lesson, {
 import { Diana } from '~/services/contributors';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.